docs/console/console-extensibility/registration.md

```diff
@@ -0,0 +1,232 @@
+---
+id: registration
+title: Register Extensions
+sidebar_label: Register Extensions
+---
+# Register Extensions
+
+A new extension always should be registered on Console before it can be activated. The registered extension are owned by specific Companies and so the registration and management operations on extensions can only be done by a Company Owner.
```

Comment:
```suggestion
Before [activating an extension](LINK TO ACTIVATION PAGE), you must register it on Console. The registered extension are owned by specific Companies and so the registration and management operations on extensions can only be done by a Company Owner.
```

Reply:
in my opinion this link on activation only serves to distract the user, because i expect that the user is here to register his extension and could interpret that redirect as something that should be read before to continue with registration

Reply:
Well this depends on your reading behaviour. If I'm reading this and I am curious about what does it mean to activate an extension I've an immediate link to that.

Also cross-linking is useful for SEO, I'd not miss the opportunity for linking pages together.